### DISCUSSION TITLE
API to export application data

### DISCUSSION THREAD

**Discussion by dararthur** [nuxsmin/sysPass#1710] [Q&A]
Hi,

I've just tried to create a syspass API in order to export all my data in an xml document but nothing's working. 
I don't know what's wrong with my code. 
Here it is :

```
<?php

define('APP_ROOT', __DIR__);
define('APP_MODULE', 'api');

require APP_ROOT . DIRECTORY_SEPARATOR . 'lib' . DIRECTORY_SEPARATOR . 'Base.php';

$json = [
    'jsonrpc' => '2.0',
    'method' => 'export',
    'params' => [
        'authToken' => 'my token here',
    ],
    'id' => 1
];
json_encode($json));
json_decode($json, true));
?>

```
If someone can help me ! 
Thanks in advance 😃 

Regards

**Comment by nuxsmin** [comment]
Hi,

the export API feature behavior is like the UI one, that's you will get a XML file within sysPass server, but not for downloading through the API.

Regards
